integration-cli/docker_cli_swarm_test.go

```diff
@@ -210,19 +210,19 @@ func (s *DockerSwarmSuite) TestSwarmServiceListFilter(c *testing.T) {
 	// We search checker.Contains with `name+" "` to prevent prefix only.
 	out, err = d.Cmd("service", "ls", "--filter", filter1)
 	assert.NilError(c, err, out)
-	assert.Assert(c, strings.Contains(out, name1+" "))
-	assert.Assert(c, !strings.Contains(out, name2+" "))
-	assert.Assert(c, !strings.Contains(out, name3+" "))
+	assert.Assert(c, strings.Contains(out, name1+" "), out)
+	assert.Assert(c, !strings.Contains(out, name2+" "), out)
+	assert.Assert(c, !strings.Contains(out, name3+" "), out)
 	out, err = d.Cmd("service", "ls", "--filter", filter2)
 	assert.NilError(c, err, out)
-	assert.Assert(c, strings.Contains(out, name1+" "))
-	assert.Assert(c, strings.Contains(out, name2+" "))
-	assert.Assert(c, !strings.Contains(out, name3+" "))
+	assert.Assert(c, strings.Contains(out, name1+" "), out)
+	assert.Assert(c, strings.Contains(out, name2+" "), out)
+	assert.Assert(c, !strings.Contains(out, name3+" "), out)
```

Comment:
FWIW; I initially wanted to use `is.Contains()` here, as it prints a nicer output, but don't think there's a "not contains", so I kept it simple for now

Reply:
One thing I don't like about gotest.tools is everything is unexported and it's really difficult to just make a minor change to an otherwise good check.

Reply:
Perhaps the idea there was to reduce the public API if not needed, but worth discussing with @dnephin and @vdemeester